ferrilata-devilline/Controllers/PitchController.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Linq;
+using ferrilata_devilline.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace ferrilata_devilline.Controllers
+{
+    public class PitchController : Controller
+    {
+        [Route("api/pitches")]
+        public IActionResult Return_Pitches()
+        {
+            var re = Request;
```

Comment:
No need to create variables for this, you can access it from the if statement like Request.Headers, it will be longer, but still shorter than with variables